tests/tensor_search/test_add_documents.py

```diff
@@ -614,26 +612,6 @@ def test_add_documents_resilient_doc_validation(self):
         tensor_search.create_vector_index(
             config=self.config, index_name=self.index_name_1)
         docs_results = [
-            # handle empty dicts
-            ([{"_id": "123", "my_field": "legitimate text"},
-             {},
-             {"_id": "456", "my_field": "awesome stuff!"}],
-             [("123", "result"), (None, 'error'), ('456', 'result')]
-             ),
-            ([{}], [(None, 'error')]),
-            ([{}, {}], [(None, 'error'), (None, 'error')]),
-            ([{}, {}, {"some_dict": "yep"}], [(None, 'error'), (None, 'error'), (None, 'result')]),
-            # handle invalid dicts
-            ([{"this is a set, lmao"}, "this is a string", {"some_dict": "yep"}], [(None, 'error'), (None, 'error'), (None, 'result')]),
-            ([1234], [(None, 'error')]), ([None], [(None, 'error')]),
-            # handle invalid field names
-            ([{123: "bad"}, {"_id": "cool"}], [(None, 'error'), ("cool", 'result')]),
-            ([{"__chunks": "bad"}, {"_id": "1511", "__vector_a": "some content"}, {"_id": "cool"},
-              {"_id": "144451", "__field_content": "some content"}],
-             [(None, 'error'), ("1511", 'error'), ("cool", 'result'), ("144451", "error")]),
-            ([{123: "bad", "_id": "12345"}, {"_id": "cool"}], [("12345", 'error'), ("cool", 'result')]),
-            ([{None: "bad", "_id": "12345"}, {"_id": "cool"}], [("12345", 'error'), ("cool", 'result')]),
-            # handle bad content
```

Comment:
Sorry, it doesn't seem I added some updates to the PR description. Because we validate immediately after the HTTP request comes in, there are some validations (generally just Typing validation) that cannot be caught and then returned as an error. These are those tests. May be a problem with this change.

Reply:
Is there any way around this? Like doing pydantic validation on each doc individually? and then adding the relevant "doc failed to be indexed" message to the response?